doc/field_guide__variable.md

```diff
@@ -0,0 +1,28 @@
+## Variable stars (variable)
+Variable stars are objects whose brightness is varying at a statistically significant level.
+ZTF has statistical measurements down to ~10 millimagnitudes.

```

Comment:
Say that this depends on the magnitude: 10 mmag for bright stars, could be >0.1 mag for fainter objects.